data-model-for-portal/Microsoft.Compute/virtualMachines.json

```diff
@@ -167,6 +242,204 @@
+            "path": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}",
+            "confirmation": false,
+            "help": {
+                "learnMore": {
+                    "url": "https://docs.microsoft.com/cli/azure/vm/nic#az-vm-nic-list"
+                },
+                "parameterSets": [
+                    {
+                        "parameters": [
+                            "--resource-group",
+                            "--vm-name"
+                        ]
+                    }
+                ]
+            },
+            "examples": [
+                {
+                    "description": "List all of the NICs on a VM.",
+                    "parameters": [
+                        {
+                            "name": "--subscription",
+                            "value": "[path.subscriptionId]"
+                        },
+                        {
+                            "name": "--resource-group",
+                            "value": "[path.resourceGroupName]"
+                        },
+                        {
+                            "name": "--vm-name",
+                            "value": "[path.vmName]"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "az vm identity show",
+            "description": "Display VM's managed identity info.",
+            "path": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}",
+            "confirmation": false,
+            "help": {
+                "learnMore": {
+                    "url": "https://docs.microsoft.com/cli/azure/vm/identity#az-vm-identity-show"
+                },
+                "parameterSets": [
+                    {
+                        "parameters": [
+                            "[--ids]",
+                            "[--name]",
+                            "[--resource-group]",
+                            "[--subscription]"
+                        ]
+                    }
+                ]
+            },
+            "examples": [
+                {
+                    "description": "display VM's managed identity info.",
+                    "parameters": [
+                        {
+                            "name": "--subscription",
+                            "value": "[path.subscriptionId]"
+                        },
+                        {
+                            "name": "--name",
+                            "value": "[path.vmName]"
+                        },
+                        {
+                            "name": "--resource-group",
+                            "value": "[path.resourceGroupName]"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "az vm aem verify",
```

Comment:
Remove this command

Reply:
Received, already modified.

Reply:
The url displayed by --debug will be used, and thus will not be modified.